emulated-integration-tests/src/chains/pop_network/genesis.rs

```diff
@@ -0,0 +1,55 @@
+// Substrate
+use sp_core::storage::Storage;
+
+// Cumulus
+use emulated_integration_tests_common::{
+	accounts, build_genesis_storage, collators, SAFE_XCM_VERSION,
+};
+use parachains_common::Balance;
+
+pub const PARA_ID: u32 = 909;
```

Comment:
Could you set this const in the chain_spec.rs? And then in the chain_spec.rs update the IDs to use that same constant

Reply:
Same with safe xcm version?

Reply:
Reset these to local consts as it meant the emulated integration tests were building the node just to get two values, which seems unnecessary especially as these emulated tests use runtimes only.